Barotrauma/BarotraumaClient/ClientSource/Screens/NetLobbyScreen.cs

```diff
@@ -2275,6 +2275,7 @@ public static void CreateModerationContextMenu(Client client)
             options.Add(new ContextMenuOption("Rank", isEnabled: canManagePermissions, options: rankOptions.ToArray()));
 
             Color clientColor = client.Character?.Info?.Job.Prefab.UIColor ?? Color.White;
+            if (GameMain.NetworkMember.ServerSettings.DisableCrewList == true) { clientColor = Color.White; }
```

Comment:
A minor note, but there's no reason to check equality to true here, `if (GameMain.NetworkMember.ServerSettings.DisableCrewList)` is enough.

What this line here is doing is essentially doing the check twice, "if it is true that the value of this variable equals to true", when it could be "if the value of this variable is true".

Applies to most of the other checks for `DisableCrewList` in this PR.